.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -147,3 +147,33 @@ steps:
         env:
           CLIMACOMMS_DEVICE: "CUDA"
           LONGER_RUN: ""
+
+  - wait
+
+  - group: "ILAMB leaderboard"
+    if: build.env("LONGER_RUN") != null && build.env("ILAMB_RUN") != null
```

Comment:
Could you add some documentation somewhere explaining how/when to set each of these? I think LONGER_RUN is pretty self-explanatory, but it's still good to note the difference it makes. and I think ILAMB_RUN is a little less clear - does setting that only determine whether the leaderboard gets generated?